.landscape.yaml

```diff
@@ -7,6 +7,7 @@ ignore-paths:
 test-warnings: true
 python-targets:
     - 3
+max-line-length: 200
```

Comment:
is this now official? in the past i tried to break after 120, but i'd love to be allowed to use 200 :)